pkg/cache/fake/objectnotifier.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The kpt and Nephio Authors
+// Copyright 2022. 2024 The kpt and Nephio Authors
```

Comment:
should this be a comma instead of a period?

Reply:
Yes, it's fixed now.